package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@rhds/icons",
+  "version": "1.0.0",
+  "description": "Red Hat Icons",
+  "main": "icons.js",
+  "module": "icons.js",
+  "types": "icons.d.ts",
+  "type": "module",
+  "scripts": {
+    "build": "echo \"no build yet\"",
```

Comment:
**PLEASE HOLD**

todo:

- [ ] optimize source svgs
- [ ] add SVGO config to automate above